src/clj/puppetlabs/services/jruby_pool_manager/jruby_schemas.clj

```diff
@@ -59,14 +59,17 @@
         values are `:jit`, `:force`, and `:off`.  Defaults to `:off`.
 
     * :max-active-instances - The maximum number of JRubyInstances that
-        will be pooled."
+        will be pooled.
+
+    * :environment-vars - The whitelisted environment variables to be passed"
   {:ruby-load-path [schema/Str]
    :gem-home schema/Str
    :compile-mode SupportedJRubyCompileModes
    :borrow-timeout schema/Int
    :max-active-instances schema/Int
    :max-borrows-per-instance schema/Int
-   :lifecycle LifecycleFns})
+   :lifecycle LifecycleFns
+   (schema/optional-key :environment-vars) {schema/Str schema/Str}})
```

Comment:
Right now everything in this config is required, and we don't validate this config until after we set a bunch of defaults - so things that are optional have a default in the `initialize-config` function (https://github.com/puppetlabs/jruby-utils/pull/17/files#diff-08581f068a6d2fb016f433b8ca45464eR131). This is kind of confusing and maybe not the best way of doing things, but I think for now it would be good for this new setting to be consistent - it should be required here and there should be a default in `initialize-config`.